src/typescript/13-utility-types/utility-types.md

```diff
@@ -27,16 +27,18 @@ interface Dinosaur {
 }
 ```
 
-## Partial&lt;Type&gt;
+## Objective: Utility Types
+
+### Partial&lt;Type&gt;
 
 All properties of `Type`, but optional.
-Partial is often used when you need to partially update an object. See `example` below.
+Partial is often used when you need to partially update an object. See the `example` below.
```

Comment:
`example` is odd because there’s no variable `example`, is it trying to point at line 10? I would use the highlighting below.